src/components/Category/Category.js

```diff
@@ -1,14 +1,30 @@
-import React from "react";
+import React, { Component } from "react";
+import PropTypes from 'prop-types';
+
+import "./../Button/button.css";
 import "./category.css";
 
-const Category = ({ name }) => (
-  <div>
-    <label className="container">
-      {name}
-      <input type="checkbox" />
-      <span className="checkmark" />
-    </label>
-  </div>
-);
```

Comment:
I'm not sure about code conventions agreed on the project but functional components looks better in this case, it more concise and has a better readability. Just my point of view :) Generally you could start with functional components and if you understand you need some state management or lifecycle methods you could always rewrite it as a class component.